pkg/resolver/search.go

```diff
@@ -133,6 +143,26 @@ func (s *SearchResult) resolveCount() (int, error) {
 	return count, err
 }
 
+func (s *SearchResult) resolveUids() ([]*string, error) {
+	rows, err := s.pool.Query(context.Background(), s.query, s.params...)
+
+	if err != nil {
+		klog.Errorf("Error resolving query [%s] with args [%+v]. Error: [%+v]", s.query, s.params, err)
```

Comment:
I wonder if we are able to process anything else when we have an error here or if we should stop and return the error.

Reply:
Do you mean for this specific resolver or for all of the resolvers?

Reply:
It applies in other places too.  I don't think `rows` will have anything if there's an error, maybe we could get partial results?